content/docs/2.9/scalers/etcd.md

```diff
@@ -0,0 +1,117 @@
++++
+title = "Etcd"
+availability = "v2.9+"
+maintainer = "Community"
+description = "Scale applications based on an etcd key-value pair. By watching an etcd key, a passively received push mode, the scaler can activate applications with lower load usage than frequent pull mode"
+go_file = "etcd_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `etcd_scaler` trigger that scales based on an etcd key-value pair.
+
+```yaml
+  triggers:
+    - type: etcd
+      metadata:
+        etcdEndpoints: 172.0.0.1:2379,172.0.0.2:2379,172.0.0.3:2379
+        etcdKey: length
```

Comment:
Isn't `key` enough?

Reply:
I'd say that we can remove the prefix `etcd` from both

Reply:
It's been fixed!

Reply:
Their prefixes were removed.